test/index.js

```diff
@@ -55,27 +63,52 @@ test('rm', t => {
   })
 })
 
+test('ls with no options', t => {
+  const roster = {
+    zkat: 'developer',
+    iarna: 'admin',
+    isaacs: 'owner'
+  }
+  tnock(t, REG).get('/-/org/myorg/user').reply(200, roster)
+  return org.ls('myorg').then(res => {
+    t.deepEqual(res, roster, 'got back a roster')
+  })
+})
+
 test('ls', t => {
   const roster = {
-    'zkat': 'developer',
-    'iarna': 'admin',
-    'isaacs': 'owner'
+    zkat: 'developer',
+    iarna: 'admin',
+    isaacs: 'owner'
   }
   tnock(t, OPTS.registry).get('/-/org/myorg/user').reply(200, roster)
   return org.ls('myorg', OPTS).then(res => {
     t.deepEqual(res, roster, 'got back a roster')
   })
 })
 
+test('ls stream with no options', t => {
+  const roster = {
+    zkat: 'developer',
+    iarna: 'admin',
+    isaacs: 'owner'
+  }
+  const rosterArr = Object.keys(roster).map(k => [k, roster[k]])
+  tnock(t, REG).get('/-/org/myorg/user').reply(200, roster)
+  return org.ls.stream('myorg').then(res => {
+    t.deepEqual(res, rosterArr, 'got back a roster, in entries format')
+  })
+})
+
 test('ls stream', t => {
   const roster = {
-    'zkat': 'developer',
-    'iarna': 'admin',
-    'isaacs': 'owner'
+    zkat: 'developer',
+    iarna: 'admin',
+    isaacs: 'owner'
   }
   const rosterArr = Object.keys(roster).map(k => [k, roster[k]])
   tnock(t, OPTS.registry).get('/-/org/myorg/user').reply(200, roster)
-  return getStream.array(org.ls.stream('myorg', OPTS)).then(res => {
+  return org.ls.stream('myorg', OPTS).then(res => {
```

Comment:
Fixes the test breaking because `.stream()` returns a minipass stream. We can just call `.collect()` in the test, rather than in the function.

```suggestion
  return org.ls.stream('myorg', OPTS)
    .collect()
    .then(res => {
```